### PR TITLE
[AKS] Migrate aks-preview module to track 2 SDK

### PR DESCRIPTION
**Pull request by FumingZhang**
Migrate aks-preview module to track 2 SDK.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
AKS

**Review by andyliuliming — APPROVED** [review]

File: src/aks-preview/azext_aks_preview/__init__.py [hunk 1]
```diff
@@ -16,7 +16,11 @@ class ContainerServiceCommandsLoader(AzCommandsLoader):
 
     def __init__(self, cli_ctx=None):
         from azure.cli.core.commands import CliCommandType
-        register_resource_type('latest', CUSTOM_MGMT_AKS_PREVIEW, '2021-05-01')
+        register_resource_type(
+            "latest",
+            CUSTOM_MGMT_AKS_PREVIEW,
+            SDKProfile("2021-05-01", {"container_services": "2017-07-01"}),
```
Comment (zhoxing-ms) [inline src/aks-preview/azext_aks_preview/__init__.py:22]: May I ask why this api-version for `container_services` uses an older one?

Comment (FumingZhang) [reply src/aks-preview/azext_aks_preview/__init__.py:22]: The "container_services" operations are only used to implement ```aks get-version``` and not updated for a long time (which is the same in azure-cli). Other ```aks xxx``` operations are supported by "managed_clusters" and "agent_pools" operations.

Comment (zhoxing-ms) [reply src/aks-preview/azext_aks_preview/__init__.py:22]: OK, got it~

**Review by zhoxing-ms — APPROVED** [review]

**Review by jsntcy — APPROVED** [review]
:shipit:

**Pull request merged**